libutils/json-utils.c

```diff
@@ -0,0 +1,263 @@
+#include <json-utils.h>
+#include <logging.h>    // Log()
+#include <file_lib.h>   // safe_fopen()
+#include <string_lib.h> // TrimWhitespace()
+#include <csv_parser.h>
+#include <alloc.h>
+#define CF_BUFSIZE 4096
+
+/**
```

Comment:
Nice helper functions in this file, which could (should?) be unit-checked very nicely. But how,  if they are internal?

FYI a common method for unit testing `static` functions is including the `.c` file on top of the unit test file. :-) For reasons, this is even better many times than exporting the function in a header file and linking to the whole library.